src/modlist.cpp

```diff
@@ -1236,17 +1240,16 @@ void ModList::removeRowForce(int row, const QModelIndex &parent)
   m_Profile->cancelModlistWrite();
   beginRemoveRows(parent, row, row);
   ModInfo::removeMod(row);
-  endRemoveRows();
   m_Profile->refreshModStatus();  // removes the mod from the status list
+  endRemoveRows();
```

Comment:
This moved down a line, was it intended?

Reply:
Nope, I don't know how it ended-up here... thanks.

Reply:
This is actually required by `endRemoveRows` will trigger a rebuild of the tree for collapsible separators, so the profile needs to be up-to-date.